iroh-gossip/src/proto/topic.rs

```diff
@@ -171,12 +171,26 @@ impl<PI: Clone> IO<PI> for VecDeque<OutEvent<PI>> {
     }
 }
 /// Protocol configuration
-#[derive(Clone, Default, Debug)]
+#[derive(Clone, Debug)]
 pub struct Config {
     /// Configuration for the swarm membership layer
     pub membership: hyparview::Config,
     /// Configuration for the gossip broadcast layer
     pub broadcast: plumtree::Config,
+    /// Max message size in bytes
```

Comment:
```suggestion
    /// Max message size in bytes.
```

just to make Floris happy